mne/inverse_sparse/mxne_optim.py

```diff
@@ -257,6 +261,9 @@ def dgap_l21(M, G, X, active_set, alpha, n_orient):
     return gap, p_obj, d_obj, R
 
 
+@deprecated(extra="Starting version v0.24, MxNE problems will be solved "
```

Comment:
That will start in 0.25 (0.24 is the upcoming release, where it will not yet be (B)CD-only)